src/main/webapp/index.html

```diff
@@ -11,7 +11,7 @@
  <body>
     <div id="content">
       <div>
-        <h1 class="heading-font">HOUSEHOLD</h1>
+        <h1 class="heading-font">houseHOLD</h1>
         <h3 class="body-font">Please log in here:<h3>
```

Comment:
I think this should be a div (maybe with class=login-prompt) instead. It's not a header (in that it's not really organizing content).